openwisp_utils/admin_theme/context_processor.py

```diff
@@ -29,7 +34,10 @@ def build_menu(request=None):
         url = reverse('admin:{}_{}_changelist'.format(app_label,
                                                       model.lower()))
         label = item.get('label', model_class._meta.verbose_name_plural)
-        model_admin = site._registry[model_class]
+        try:
+            model_admin = admin_site._registry[model_class]
+        except Exception:  # pragma: no cover
```

Comment:
again, why use a generic exception? How can it fail?

I think we can register into both our admin_site and the default django one for backward compatibility and add a comment like:
```
# TODO: remove once all openwisp modules have been upgraded
```
above the line which is kept for backward compatibility.

Reply:
@nemesisdesign Okay, I made some changes, please checkout